src/index.html

```diff
@@ -5,8 +5,334 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__link">
+        <img
+          class="header__logo logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__li">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__li">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__card card" data-qa="card">
+        <a class="card__image" href="/">
+          <img
+            class="card__imac"
```

Comment:
It's not a good idea to have product name as a class as products can be different but the HTML element stays the same

Reply:
Not fixed. Just name it e.g. 'card__image' and link above 'card__image-link', so your BEM logic stays the same even with a completely different product in the card, not only iMac.